### PR TITLE
Svd09 patch r proc 3

### PR DESCRIPTION
**Pull request by rivera-lanasm**
This branch contains the Luigi tasks associated with calculating RT and DT (only for Mumbai currently) using input data from S3.

 @svd09 the R_scripts/doub_time_test.R script contains a different implementation of the DT code. I believe this is equivalent to the implementation in dt_script.R, but I want to check with you. 

I will create another branch based on this one, wherein I will complete implementation of RT district filtering (critical cities), implementing DT for all districts, and then implementing RT and DT for Mumbai wards as well. 

Currently the process outputs to S3 path: the-covid-lens/Phase2_DT.csv and the-covid-lens/Phase2_RT.csv. I am wondering if it could make more sense to output distinct RT and DT files for each district, although I'm not sure what the dashboard currently expects to ingest.

### CODE REVIEW

**Comment by svd09** [comment]
Thanks. For Rt that is fine. For DT we are still debating which DT function to use. There is a python DT function that we may use. So, would suggest doing that only for Rt right now.

**Review by Nozziel — COMMENTED** [review]
phase2_rt_df seems a bit behind with no aws stuff.

**Review by Nozziel — APPROVED** [review]
dont forget to update the readme.md as what github actions now belong to phase 2 (next to the python one)

File: backend/metrics/calculations.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import pandas as pd
 from numpy import random
 import numpy as np
-
+from typing import List
```
[the remark this answers is not in the document]
Comment (rivera-lanasm) [reply backend/metrics/calculations.py:4]: for some reason when I run this job using "list[str]", I run into this error:
 TypeError: 'type' object is not subscriptable

being run in the context of 
"/home/runner/work/etl-pipeline/etl-pipeline/backend/metrics/calculations.py", line 48    
def _moving_average_grouped(df: pd.DataFrame, group_columns: list[str], target_column: str, window_size) -> pd.Series:

I don't run into the error when I use "List[str]"

process log:
https://github.com/swb-ief/etl-pipeline/runs/2104732402?check_suite_focus=true

**Pull request closed without merging**